### ISSUE TITLE
Selcted value is not changes if I give preselected value

### ISSUE THREAD

**Issue by dnsoumik** [icemanbsi/searchable_dropdown#129]
onChange function I am always getting the preselected value not the value user clicked

**Comment by HetalGohil** [comment]
yes i am also getting first value when selected on onchage.in onchange it is not updating value

**Comment by isiyemijoe** [comment]
Same issue here any fix yet?

**Comment by lcuis** [comment]
Hello @dnsoumik , @HetalGohil and @isiyemijoe ,

Thanks for taking the time to write here.

Can you please give a minimal yet complete and runnable example of what you mean?

**Comment by nagasaidurga** [comment]
Yes Same issue for me as well. If we add preselected option with value parameter with SearchableDropdown constructor.Then its always giving the same value in the onchange method.


Please fix this issue as early as possible.

**Comment by PhilDevs94** [comment]
> onChange function I am always getting the preselected value not the value user clicked

Could you show specific your code ?

**Comment by shady11** [comment]
same issue, any updates?
